datasets/simulations/README.rst

```diff
@@ -0,0 +1,4 @@
+This contains a `MapDataset` and two `SpectrumDatasetOnOff` created through simulations. The former utilises the cta-1dc South file supplied with GAMMAPY_DATA.
+
```

Comment:
Could you precise here that the Gammapy v2.0 environment is used to make these datasets?